### PR TITLE
evebox: init at 0.17.2

### LINKED ISSUE #313866: Package request: EveBox
**Project description**

EveBox is a Suricata alert and event management tool for the Suricata IDS/NSM Engine.

**Metadata**

* homepage URL: https://evebox.org/
* source URL: https://github.com/jasonish/evebox
* license: mit
* platforms: unix, linux, darwin, ...

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by felbinger**
## Description of changes

EveBox is a Suricata alert and event management tool for the Suricata IDS/NSM Engine.

Closes #313866

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by felbinger** [comment]
Result of `nixpkgs-review pr 313896` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)

<details>
<summary>1 package built:</summary>
<ul>
<li> evebox
</ul>
</details>

File: pkgs/by-name/ev/evebox/package.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  src = fetchFromGitHub {
+    owner = "jasonish";
+    repo = "evebox";
+    rev = version;
+    hash = "sha256-djL5cdudJNPAWLMQPS2Dkcc9H/gouOuu8evcBDdY9wA=";
+  };
+
+  cargoLock = {
+    lockFile = "${src}/Cargo.lock";
```
Comment (pbsds) [inline pkgs/by-name/ev/evebox/package.nix:18]:
```suggestion
    lockFile = ./Cargo.lock;
```

This is IFD, which is disallowed in nixpkgs. Please vendor the lock file

Comment (felbinger) [reply pkgs/by-name/ev/evebox/package.nix:18]: Funny, I already had it fixed before... (ofborg already informed me regarding this). I guess I force pushed from a wrong device two days ago... :joy:

**Comment by felbinger** [comment]
I had this same issue exactly two weeks ago and I do not remember what the solution was...

File: pkgs/by-name/ev/evebox/package.nix (hunk 1 above)
Comment (pbsds) [reply pkgs/by-name/ev/evebox/package.nix]: `--force-with-lease` ;)

**Comment by pbsds** [comment]
can you set `meta.broken = stdenv.isDarwin;`?

File: pkgs/by-name/ev/evebox/package.nix (hunk 1 above)
Comment (felbinger) [reply pkgs/by-name/ev/evebox/package.nix]: Oooh, that's brilliant, I didn't even know that option existed  :exploding_head: Thanks!

**Pull request merged**